Source/Core/Core/HW/GCMemcard/GCMemcard.cpp

```diff
@@ -30,22 +30,14 @@ static void ByteSwap(u8* valueA, u8* valueB)
   *valueB = tmp;
 }
 
-GCMemcard::GCMemcard(const std::string& filename, bool forceCreation, bool shift_jis)
-    : m_valid(false), m_filename(filename)
+GCMemcard::GCMemcard(const std::string& filename) : m_valid(false), m_filename(filename)
 {
   // Currently there is a string freeze. instead of adding a new message about needing r/w
```

Comment:
This comment seems heavily outdated...

Reply:
Yeah I noticed that too. I think I want to redesign the entire way this is constructed anyway due to the PanicAlerts so this will probably go away with that.